### PR TITLE
SDNTB-700 ADD: Implement FTP ingest service in planning.

### PR DESCRIPTION
**Pull request by dev-tsoni**

### CODE REVIEW

**Review by petrjasek — APPROVED** [review]

**Comment by petrjasek** [comment]
btw would the standard file feed ingest work now with events json?

**Comment by MarkLark86** [comment]
> btw would the standard file feed ingest work now with events json?

It will work with Events JSON, but It won't for the ICS parser. Will need some changes to File, HTTP & Email Feeding Services in core, then we should be able to remove those Event specific services from Planning

**Comment by dev-tsoni** [comment]
> > btw would the standard file feed ingest work now with events json?
> 
> It should work with Events JSON, but It won't for the ICS parser. Will need some changes to File, HTTP & Email Feeding Services in core, then we should be able to remove those Event specific services from Planning

We can add these changes altogether in single ticket.

**Comment by petrjasek** [comment]
lets do the ftp first, that's what we need now for qa

File: superdesk/io/commands/update_ingest.py [hunk 1]
```diff
@@ -697,7 +697,7 @@ def get_ingest_collection(feeding_service, item):
         ingest_collection = feeding_service.service
 
     # If the type of item is event, set the collection to events
-    elif item.get("type") and item.get("type") == "event":
+    elif item.get(ITEM_TYPE) and item.get(ITEM_TYPE) == "event":
```
Comment (petrjasek) [inline superdesk/io/commands/update_ingest.py:700]: imo `elif item.get(ITEM_TYPE) == "event"` would be enough

**Review by petrjasek — APPROVED** [review]

**Pull request merged**